src/core/graphics/webgl/utils/buildPoly.js

```diff
@@ -51,8 +51,6 @@ var buildPoly = function (graphicsData, webGLData)
 
         var vertPos = verts.length / 6;
 
-        var i = 0;
```

Comment:
Instead of relying on function-scope hoisting for the above `var i` can we just move its decl to the top of the function for clarity?